app/lib/features/chat/widgets/video_message_builder.dart

```diff
@@ -116,24 +116,72 @@ class VideoMessageBuilder extends ConsumerWidget {
     BuildContext context,
     MediaChatState mediaState,
   ) {
-    return ClipRRect(
-      borderRadius:
-          isReplyContent ? BorderRadius.circular(6) : BorderRadius.circular(15),
-      child: SizedBox(
-        height: 150,
-        child: ActerVideoPlayer(
-          videoFile: mediaState.mediaFile!,
-          onTapFullScreen: () {
-            showAdaptiveDialog(
-              context: context,
-              barrierDismissible: false,
-              useRootNavigator: false,
-              builder: (ctx) => VideoDialog(
-                title: message.name,
-                videoFile: mediaState.mediaFile!,
+    return InkWell(
+      onTap: () {
+        showAdaptiveDialog(
+          context: context,
+          barrierDismissible: false,
+          useRootNavigator: false,
+          builder: (ctx) => VideoDialog(
+            title: message.name,
+            videoFile: mediaState.mediaFile!,
+          ),
+        );
+      },
+      child: ClipRRect(
```

Comment:
I feel like this is nesting deep enough now to be moved into its own private Widget in this file for lower indentation levels and better legibility.